specs/tile_bag_spec.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'spec_helper'
+
+describe "Tilebag" do
+
+  before do
+    @tilebag = Scrabble::Tilebag.new
+  end
+
+  describe "#initialize" do
+
+    it "should have 98 tiles in @tilebag" do
+      @tilebag.tiles.length.must_equal 98
+      @tilebag.tiles.must_be_kind_of Array
+      @tilebag.must_respond_to :tiles
+    end
+
+  end
+
+  describe "#draw_tiles" do
+
+    it "should return an array of picked _tiles" do
+      @tilebag.draw_tiles(7).must_be_kind_of Array
+    end
+
+    it "when drawing one tile should return one tile" do
+      picked_tiles = @tilebag.draw_tiles(1)
+      picked_tiles.length.must_equal 1
+    end
+
+    it "will return more than one tile" do
+      picked_tiles = @tilebag.draw_tiles(98)
+      picked_tiles.length.must_equal 98
+    end
+
+    it "should remove each tile from the @tile" do
+      starting_tiles = @tilebag.tiles.length
+      num = 10
+      @tilebag.draw_tiles(num)
+      @tilebag.tiles.length.must_equal starting_tiles - num
+    end
+
```

Comment:
You're missing a couple of interesting edge cases here:
- What happens if you attempt to draw 0 tiles?
- What happens if you attempt to draw the last tile from the bag?
- What happens when you attempt to draw more tiles than are in the bag?